zerver/openapi/zulip.yaml

```diff
@@ -3362,7 +3362,19 @@ paths:
               example: [9, 10]
           required: true
         - $ref: "#/components/parameters/RequiredContent"
-        - $ref: "#/components/parameters/Topic"
+        - name: topic
+          in: query
+          description: |
+            The topic of the message. Only required for stream messages
+            (`type="stream"`), ignored otherwise.
+
+            Maximum length of 60 characters.
+
+            **Changes**: New in Zulip 2.0.  Previous Zulip releases encoded
+            this as `subject`, which is currently a deprecated alias.
+          schema:
+            type: string
+          example: Castle
```

Comment:
For the TODO, I would expect that `update_message_backend` ignores that parameter, but a test is a good idea.  I think the reason we don't ignore it is that we may later introduce support for topics in PM threads; perhaps we should mention that future possibility in this documentation.

Reply:
Also, can you comment in the commit message on in what way these two `topic` entries are different?

Reply:
You said you would expect that it ignores `topic`, but then said "the reason we don't ignore it is ...". Or did you mean you _would_ expect it to ignore `topic`, and the reason it _may not be_ is PM threads?

I will definitely be expanding the commit message.